src/scripts.js

```diff
@@ -0,0 +1,225 @@
+// News API key: 9ed005ef4eb94baf913fce701c69972f
+
+//TODO: stylization, excludeDomains/blocklist
+
+//Object contains possible URLS for fetch requests
+const apiRequests = {
```

Comment:
Very nice. I like how you have created an object that encapsulates all URL generation logic in one place